### DISCUSSION TITLE
Using model-viewer in Android app

### DISCUSSION THREAD

**Discussion by Kartofanych** [google/model-viewer#3956] [Q&A]
I am trying to implement this solution to see my 3d models in Android app throw webview. For me it is important to keep my 3d models locally, but when I try linking to them locally, I got this exception and model does not download: "Fetch API cannot load file:///android_asset/hey.glb. URL scheme "file" is not supported." Is there any way for me to keep them local?

**Comment by elalish** [comment]
Maybe this? https://stackoverflow.com/questions/10620417/access-local-file-in-a-webview
